### ISSUE TITLE
Assert should be preceded and followed by a blank line

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#331]
A call to `Assert` should be preceded by a blank line if the preceding line contains a call to something that is no `Assert`.
The reason is ease of reading (spotting asserts with ease).

Following should report a violation:
```c#
var x = 42;
var y = "something";
Assert.That(x, Is.EqualTo(42));
Assert.That(y, Is.EqualTo("something"));
```

While following should **not** report a violation:
```c#
var x = 42;
var y = "something";

Assert.That(x, Is.EqualTo(42));
Assert.That(y, Is.EqualTo("something"));
```

**Comment by RalfKoban** [comment]
In addition, a call to `Assert` should be followed by a blank line:

Following should report a violation:
```c#
Assert.That(x, Is.EqualTo(42));
Assert.That(y, Is.EqualTo("something"));
var x = 42;
var y = "something";
```

While following should **not** report a violation:
```c#
Assert.That(x, Is.EqualTo(42));
Assert.That(y, Is.EqualTo("something"));

var x = 42;
var y = "something";
```

**Comment by RalfKoban** [comment]
Same is valid for `Assume`.
